apps/frontend/src/libs/components/toast-container/toast-container.tsx

```diff
@@ -0,0 +1,24 @@
+import { ToastContainer as LibraryToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import styles from "./styles.module.css";
+
+const ToastContainer: React.FC = () => {
+	return (
+		<LibraryToastContainer
+			autoClose={5000}
+			className={styles["toast-container"] ?? ""}
+			closeOnClick
+			draggable
```

Comment:
Add explicit true or false attributes to all boolean properties

Reply:
This cause linter error:
![image](https://github.com/user-attachments/assets/fec68cae-648a-46b6-9f2d-a62c68c8f6f9)